project/Dependencies.scala

```diff
@@ -1,14 +1,13 @@
 import sbt._
 
 object Dependencies {
-  lazy val http4sVersion = "0.21.0-M5"
 
   lazy val scalatest = "org.scalatest" %% "scalatest"        % "3.1.1"
   lazy val scalameta = "org.scalameta" %% "scalameta"        % "4.3.0"
-  lazy val scalafmt  = "org.scalameta" %% "scalafmt-dynamic" % "2.5.0-RC2"
+  lazy val scalafmt  = "org.scalameta" %% "scalafmt-dynamic" % "2.6.4"
 
-  lazy val enumaratum      = "com.beachape"           %% "enumeratum"              % "1.5.13"
-  lazy val enumeratumCirce = "com.beachape"           %% "enumeratum-circe"        % "1.5.22"
+  lazy val enumeratum      = "com.beachape"           %% "enumeratum"              % "1.6.1"
+  lazy val enumeratumCirce = "com.beachape"           %% "enumeratum-circe"        % "1.6.1"
   lazy val circeRefined    = "io.circe"               %% "circe-refined"           % "0.13.0"
```

Comment:
shouldn't we extract `val`s like `circeVersion` for `circeRefined`/`circeCore`...?